# Defines the position on the map in world coordinate system
#
# Part of the state message

# X-position of the AGV in meters on the map in world coordinate system
# Precision is implementation-specific
#
# Field type: Required
float64 x

# Y-position of the AGV in meters on the map in world coordinate system
# Precision is implementation-specific
#
# Field type: Required
float64 y

# Orientation of the AGV in radians
#
# Range: [-PI, PI]
# Field type: Required
float64 theta

# Unique ID of the map in which the position is referenced
#
# Field type: Required
string map_id

# Additional information about the map
#
# Field type: Optional
string[<=1] map_description

# Indicates if the AGV's position is initialized
#     - True if position is initialized
#     - False if position is not initialized
#
# Field type: Required
bool position_initialized false

# Desribes the quality of localization and can be used to describe the accuracy
# of the current position information. Primarily for logging and visualization
# purposes
#     - 0.0 denotes position unknown
#     - 1.0 denotes position known
#
# Field type: Optional
float64[<=1] localization_score

# Deviation from position in meters. Optional for vehicles that cannot estimate
# their deviation
#
# Field type: Optional
float64[<=1] deviation_range
